opentelekomcloud/services/vpc/resource_opentelekomcloud_networking_router_v2.go

```diff
@@ -106,84 +103,81 @@ func resourceNetworkingRouterV2Create(ctx context.Context, d *schema.ResourceDat
 	}
 
 	externalGateway := d.Get("external_gateway").(string)
+	snat := d.Get("enable_snat").(bool)
```

Comment:
This will return `false` if it isn't set, this changes behavior

Reply:
Nope, it's okay
![image](https://user-images.githubusercontent.com/17201690/124261328-51fbb000-db39-11eb-80cc-2e376c2c473f.png)